backstop/tests/fieldset.html

```diff
@@ -0,0 +1,194 @@
+                aria-hidden="true"
+                focusable="false"
+              >
+                <circle
+                  cx="8"
+                  cy="8"
+                  r="6"
+                />
+              </svg>
+            </span>
+            <span class="iui-label">Light</span>
+          </label>
+          <label class="iui-radio">
+            <input
+              type="radio"
+              name="radio-demo-1"
+              value="dark"
+            />
+            <span class="iui-radio-dot">
+              <svg
+                viewBox="0 0 16 16"
+                aria-hidden="true"
+                focusable="false"
+              >
+                <circle
+                  cx="8"
+                  cy="8"
+                  r="6"
+                />
+              </svg>
+            </span>
+            <span class="iui-label">Dark</span>
+          </label>
+          <label class="iui-radio">
+            <input
+              type="radio"
+              name="radio-demo-1"
+              value="match-device"
+              checked
+            />
+            <span class="iui-radio-dot">
+              <svg
+                viewBox="0 0 16 16"
+                aria-hidden="true"
+                focusable="false"
+              >
+                <circle
+                  cx="8"
+                  cy="8"
+                  r="6"
+                />
+              </svg>
+            </span>
+            <span class="iui-label">Match device</span>
+          </label>
+        </div>
+
+        <label class="iui-toggle-switch">
+          <input
+            type="checkbox"
+            checked
+          />
+          <span class="iui-toggle">
+            <span class="iui-handle"></span>
+          </span>
+          <span class="iui-label">Share usage statistics</span>
+        </label>
+
+        <label class="iui-toggle-switch">
+          <input
+            type="checkbox"
+            checked
+          />
+          <span class="iui-toggle">
+            <span class="iui-handle"></span>
+          </span>
+          <span class="iui-label">Share crash logs</span>
+        </label>
+
+        <label class="iui-toggle-switch">
+          <input
+            type="checkbox"
+            disabled
+          />
+          <span class="iui-toggle">
+            <span class="iui-handle"></span>
+          </span>
+          <span class="iui-label">Advanced settings</span>
+        </label>
+
+      </fieldset>
+
+      <br />
+
+      <fieldset
+        class="iui-fieldset"
+        disabled
+      >
+        <legend>Advanced settings</legend>
+
+        <label class="iui-input-container iui-inline iui-disabled">
+          <div class="iui-label">Internal URL</div>
+          <input
+            value="http://127.0.0.1:5500/backstop/minified/fieldset.html"
+            class="iui-input"
+            disabled
+          />
+        </label>
+
+        <label class="iui-toggle-switch">
```

Comment:
Accidently saw that when you hover over green area (padding), it shows pointer cursor. 
![image](https://user-images.githubusercontent.com/36186912/118084238-08a8a300-b3c9-11eb-85e1-4b5f0ab631c4.png)
We either need to remove this padding or change it to margin.

Reply:
I tried swapping `padding` with `margin` on toggle switches but it's causing more changes than expected.
![Screen Shot 2021-05-13 at 10 10 08 AM](https://user-images.githubusercontent.com/849817/118137624-43570f00-b3ba-11eb-902e-71e8cddbea7b.png)
![Screen Shot 2021-05-13 at 10 11 01 AM](https://user-images.githubusercontent.com/849817/118137718-61bd0a80-b3ba-11eb-841c-0cdc241da87f.png)

Reply:
Ah yeah, vertical margins collapse. Created an item in our backlog. We will take a look at it later.